README.md

```diff
@@ -4,42 +4,59 @@ Provides Azure Authentication Middleware for a Laravel App.  If you like this, c
 
 ## Normal Installation
 
-1. `composer require rootinc/laravel-azure-middleware`
-2. run `php artisan vendor:publish --provider="RootInc\LaravelAzureMiddleware\AzureServiceProvider"` to install config file to `config/azure.php`
-3. In our routes folder (most likely `web.php`), add
+1. From the command line, run `composer require rootinc/laravel-azure-middleware`
+2. After Composer has installed the package, run `php artisan vendor:publish --provider="RootInc\LaravelAzureMiddleware\AzureServiceProvider"` to install the config file to `config/azure.php`
+3. Add the login and login callback routes to you router (by default `routes/web.php`) like shown below:
 ```php
-Route::get('/login/azure', '\RootInc\LaravelAzureMiddleware\Azure@azure');
-Route::get('/login/azurecallback', '\RootInc\LaravelAzureMiddleware\Azure@azurecallback');
+Route::get('/azure/login', [\RootInc\LaravelAzureMiddleware\Azure::class, 'azure'])->name('azure.login');
+Route::get('/azure/callback', [\RootInc\LaravelAzureMiddleware\Azure::class, 'azurecallback']);
 ```
 
-4. In our `App\Http\Kernel.php` add `'azure' => \RootInc\LaravelAzureMiddleware\Azure::class,` most likely to the `$routeMiddleware` array.
-5. In our `.env` add `AZURE_TENANT_ID, AZURE_CLIENT_ID, AZURE_CLIENT_SECRET and AZURE_RESOURCE`.  We can get these values/read more here: https://portal.azure.com/ (Hint: AZURE_RESOURCE should be https://graph.microsoft.com)
-6. As of 0.8.0, we added `AZURE_SCOPE`, which are permissions to be used for the request.  We can read more about these here: https://docs.microsoft.com/en-us/graph/api/resources/users?view=graph-rest-1.0
-7. We also added an optional `AZURE_DOMAIN_HINT` that can be used to help users know which email address they should login with.  More info here: https://azure.microsoft.com/en-us/updates/app-service-auth-and-azure-ad-domain-hints/
-8. Within our app on https://portal.azure.com/ point `reply url` to the `/login/azurecallback` route with the full url (ex: http://thewebsite.com/login/azurecallback).
-9. Add the `azure` middleware to your route groups on any routes that needs protected by auth and enjoy :tada:
+4. Open `App/Http/Kernel.php` and add `'azure' => \RootInc\LaravelAzureMiddleware\Azure::class,` to the `$routeMiddleware` array to register the middleware.
+5. Open `.env` and add the following variables (with your own values):
+```
+AZURE_TENANT_ID=your-azuread-tenant-id
+AZURE_CLIENT_ID=your-app-registration-application-id
+AZURE_CLIENT_SECRET=your-app-registration-client-secret
+AZURE_RESOURCE=https://graph.microsoft.com
+```
+All these values can be gotten from your tenant through https://portal.azure.com.
```

Comment:
Instead of `gotten`, how about 'obtained`?

Reply:
Fixed in latest push